src/components/Taf/TafSelector.jsx

```diff
@@ -10,15 +10,21 @@ export default class TafSelector extends PureComponent {
     const { selectableTafs, selectedTaf, onChange } = this.props;
     return <Row className='TafSelector'>
       <Col xs='auto'>TAF for</Col>
-      <Col xs='auto'>
-        <Typeahead labelKey={option => option.label.text}
+      <Col xs='6'>
+        <Typeahead
+          filterBy={(option, props) => {
+            return true;
+          }}
+          labelKey={option => option.label.text}
           options={selectableTafs} onChange={onChange}
           selected={selectedTaf || []} placeholder={'Select a TAF'}
           renderMenuItemChildren={(option, props, index) => {
             return <Row>
-              <Col xs='5'>{option.location}</Col>
-              <Col xs='5'>{option.label.time}</Col>
-              <Col xs='2'><Icon name={option.label.icon} /></Col>
+              <Col xs='1'><Icon name={option.label.icon} /></Col>
+              <Col xs='1'>{option.location}</Col>
+              <Col xs='1'>{option.label.time}</Col>
+              <Col xs='3'>{option.status}</Col>
+              <Col xs='6'>{option.uuid}</Col>
```

Comment:
UUID might not be that interesting for users, we should not bother users with this technical, random id.